corpysprofiling/corpysprofiling.py

```diff
@@ -165,23 +165,23 @@ def corpus_viz(corpus):
     # To get a data frame summary of the words and length of the words
     df = pd.DataFrame({'word': clean_corpus})
     df["length"] = df['word'].str.len()
-    
     df_length = df.sort_values(by="length", ascending=False).head(30)
-
-    df_freq = df.value_counts().reset_index().rename(columns = {0 :  "freq"})
+    df_freq = df['word'].value_counts().reset_index().rename(columns = {'word': "freq"})
+    
```

Comment:
@jkim222383 
I had to fix it this way in order to display the chart as expected. Without renaming "word" to "freq", there were two columns (an `index` column for all the words and a `word` column for frequency of the words.) + an index listed in df_freq. We don't have 0 listed there.